ports/subversion/unofficial-subversion-config.cmake

```
@@ -0,0 +1,168 @@
+include(CMakeFindDependencyMacro)
+
+find_dependency(apr CONFIG REQUIRED)

```

This will make a build fail immediately if there is a problem with apr. But it is not a decision the packages shall make. Let `find_dependency` take care of "required".
```suggestion
find_dependency(apr CONFIG)
```